docs/language/operators/README.md

```diff
@@ -0,0 +1,25 @@
+# Operators
+
+Dataflow operators process a sequence of input values to create an output sequence
+and appear as the components of a dataflow pipeline:
```

Comment:
I think this would read better with a period instead of a colon since it doesn't really introduce the list that follows.
```suggestion
and appear as the components of a dataflow pipeline.
```